rustworkx/rustworkx.pyi

```diff
@@ -552,39 +554,39 @@ def directed_gnm_random_graph(
     num_edges: int,
     /,
     seed: int | None = ...,
-) -> PyDiGraph: ...
```

Comment:
So based on the earlier comment is this actually `[Any, Any]`. The weights from the generator functions are a fixed type depending on the arguments. Like for this gnm random graph function the nodes and edges are always `None`.

Reply:
I think now if someone specifies `graph: PyDiGraph[None, None] = directed_gnm_random_graph(10, 10)`, MyPy will not complain which should